pkgs/applications/networking/cluster/hetzner-kube/default.nix

```diff
@@ -1,18 +1,20 @@
 { lib, buildGoPackage, fetchFromGitHub, ... }:
 
-let version = "0.3.1"; in
-
-buildGoPackage {
-  name = "hetzner-kube-${version}";
+buildGoPackage rec {
+  name = "${pname}-${version}";
```

Comment:
This is done default now in `stdenv.mkDerivation` and extends to `buildGoPackage`